### PR TITLE
Run dev environment

### PR DESCRIPTION
**Pull request by PashaKlybik**
Update vendor for fix bug with conection to BTC NODE
wrong read from configuration file
fix wrong certificate signed by unknown authority on docker conteiner
fix panic after wrong response on resync ns-btc

### CODE REVIEW

File: ns-btc/server.go [hunk 1]
```diff
@@ -230,6 +230,7 @@ func (s *Server) EventResyncAddress(c context.Context, address *pb.AddressToResy
 		resp, _, errs := request.Get(url).Retry(10, 10*time.Second, http.StatusForbidden, http.StatusBadRequest, http.StatusInternalServerError).End()
 		if len(errs) > 0 {
 			log.Errorf("EventResyncAddress:request.Get: %v", errs)
+			return nil, fmt.Errorf("EventResyncAddress: request.Get : %v", errs)
```
Comment (Enmk) [inline ns-btc/server.go:233]: Please use `errors.Wrapf(err, "%s Request failed", url)` here

https://godoc.org/github.com/pkg/errors

**Review by nickpro — DISMISSED** [review]

File: ns-eth/eth.go [hunk 2]
```diff
@@ -35,9 +35,10 @@ type Client struct {
 }
 
 type Conf struct {
-	Address string
-	RpcPort string
-	WsPort  string
+	Address  string
+	RpcPort  string
+	WsPort   string
+	WsOrigin string
```
Comment (Enmk) [inline ns-eth/eth.go]: Please add a comment here:
```// Overrides WS origin if non-empty.```

**Review by Enmk — CHANGES_REQUESTED** [review]
Mostly Ok, please fix highlighted issues and update vendored btcdsuite properly with:
```
govendor fetch github.com/btcsuite/btcd
```

Also please update existing docker-compose.yml (all secrets should be provided via env-variables and/or config files)

File: multy-back.go [hunk 3]
```diff
@@ -149,7 +150,7 @@ func (m *Multy) SetUserData(userStore store.UserStore, ct []store.CoinType) ([]s
 		if err != nil {
 			return servicesInfo, fmt.Errorf("SetUserData: userStore.FindUsersContractsChain: curID :%d netID :%d err =%s", conCred.СurrencyID, conCred.NetworkID, err.Error())
 		}
-		if len(usersData) == 0 {
+		if len(usersContracts) == 0 {
```
Comment (Enmk) [inline multy-back.go:153]: Why this ?

Comment (PashaKlybik) [reply multy-back.go:153]: Typo when copying was here.

File: .gitignore [hunk 4]
```diff
@@ -1,14 +1,17 @@
-cmd/cmd
-cmd/test
-cmd/stage
 cmd/**/*.config
-cmd/**/*.cert
 **/logs/*
-*/**/*.log
+*/*.log
```
Comment (Enmk) [inline .gitignore:3]: `*/**/*.log` matches all .log files, no matter how deep

File: .gitignore [hunk 5]
```diff
@@ -1,14 +1,17 @@
...
 cmd/ns-eth/ns-eth
+cmd/ns-eth/ns-eth-test
+cmd/ns-eth/ns-eth-main
+cmd/ns-eth/debug
 cmd/ns-btc/ns-btc
+cmd/ns-btc/ns-btc-main
+cmd/ns-btc/ns-btc-test
+cmd/ns-btc/debug
 cmd/multy-back/multy-back
+cmd/currencies
```
Comment (Enmk) [inline .gitignore:14]: Is that a directory with config files ?

**Review by Enmk — APPROVED** [review]
Ok

**Pull request merged**